src/smartcontracts/smartContract.ts

```diff
@@ -12,11 +12,10 @@ import { Interaction } from "./interaction";
 import { CallArguments, DeployArguments, ICodeMetadata, ISmartContract, QueryArguments, UpgradeArguments } from "./interface";
 import { NativeSerializer } from "./nativeSerializer";
 import { Query } from "./query";
-import { ArwenVirtualMachine, ContractCallPayloadBuilder, ContractUpgradePayloadBuilder } from "./transactionPayloadBuilders";
+import { ArwenVirtualMachine } from "./transactionPayloadBuilders";
```

Comment:
rename `Arwen` to something else

Reply:
marked it as deprecated and created a new one called `WasmVirtualMachine` with the same value